cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -310,7 +310,13 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
         val resolutionProvider = DefaultResolutionProvider.create(ortResultInput, resolutionsFile)
         val licenseClassifications =
             licenseClassificationsFile.takeIf { it.isFile }?.readValue<LicenseClassifications>().orEmpty()
-        val evaluator = Evaluator(ortResultInput, licenseInfoResolver, resolutionProvider, licenseClassifications)
+
```

Comment:
Commit message nits:
- "for dependency source" -> "for dependency source*s*" or "for the source code of dependencies" or similar
- "not have obvious need" -> "not have obvious need*s*" or just "is usually not required"
- "exection time" -> "exec*u*tion time"